### ISSUE TITLE
More than 64mb of RAM but less than 128mb

### ISSUE THREAD

**Issue by amerika13** [Ryzee119/OpenXenium#13]
I don't know if this is an issue, a potential feature request or just something I need to deal with, but I installed the Open Xenium chip today and I setup Xblast OS so that I could test RAM chips as I installed them. However, I came to the realization that I didn't set Xblast OS as the default kernel before installing the first chip. My Xbox no longer boots to Open Xenium.

My question is, should it boot regardless of RAM or is that not something it handles?

If it is handled by Open Xenium, I guess I have an issue with my RAM chip installation.

I doubt there is any bypass to boot Xblast OS, but if there is, let me know.

**Comment by Ryzee119** [comment]
Setting Xblast as default may have not made any difference anyway because it actually still boots XeniumOS first but then quickly switches bank and restarts seemlessly to give the appearance of direct boot.

Im not aware of how XeniumOS handles RAM over 64MB.

* Try booting with eject button to force XeniumOS.
* Try older XeniumOS versions. For example pre-version 2.0 runs a Cromwell backend so may boot with dodgy ram chips past 64MB. (Although not sure how you can downgrade now, maybe the recovery jumpers could do it with a older recovery.bin burnt to a DVD)

Unfortunately the best bet I think is a go back to a alladdin or something with Xblast or finish the install and hope for the best 👍

**Comment by amerika13** [comment]
Thank you for the response and I will be finishing the install and hope for the best haha. I've been metering it out as I go and everything checks out compared to the other factory installed chips for my first chip install.  I'll just repeat that 3 more times and hope things boot 👍 

Take care and thank you so much for your efforts on this project as it has already been extremely useful for myself and some others I know who was not aware of it until recently.

**Comment by amerika13** [comment]
I closed this, but I would like to add that I finished installing the remaining 3 chips and Open Xenium now boots up just fine. And, of course, I can now get into Xblast OS and test the chips (they tested out good).

So, for at least 1.4 motherboard Xbox's, Open Xenium works with 128MB of ram. Or at least boots up. I haven't gotten to poke around with some of the features. But my guess is that it will be just fine.

Thank you again!
